### ISSUE TITLE
Work on new theme

### ISSUE THREAD

**Issue by dholbach** [fluxcd/website#16]
fluxcd.io is suffering from https://github.com/tengbao/vanta/issues/78 - it'd be good if we could figure out a solution or use a different background / illustration.

**Comment by dholbach** [comment]
Choosing a new theme for the website, it'd be good if it'd include a blog.

**Comment by dholbach** [comment]
@hiddeco @stefanprodan @squaremo @bia @scottrigby @stealthybox @mewzherder @staceypotter Would you have any other requirements for a website theme?

I personally have some familiarity with https://github.com/google/docsy and can recommend it

**Comment by bia** [comment]
Docsy looks great, I'd also be happy to just drop the current animation in favour of something [much lighter](https://codepen.io/vaibhavarora/pen/xmpxjp).

**Comment by dholbach** [comment]
Yes, I'm all for it! :heart:

**Comment by hiddeco** [comment]
When I started hacking on the Docsy theme, I ran into multiple problems around some things not being configurable or not being to my likening design wise (the blog section for example).

This made me decide to continue iterating on the theme we already had (based on the [Bulma CSS framework](https://bulma.io)), while taking the Docsy theme as an inspiration for some of the things that are great about it (the documentation section for example). The work in progress for this can be found in the [`next-gen` branch](https://github.com/fluxcd/fluxcd.io/tree/next-gen).
